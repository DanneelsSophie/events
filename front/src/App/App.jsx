import PropTypes from 'prop-types';
import { QueryClientProvider, QueryClient } from 'react-query';

import Events from '../Events';

const queryClient = new QueryClient();
const App = ({ EventsCmpt }) => (
  <QueryClientProvider client={queryClient}>
    <EventsCmpt />
  </QueryClientProvider>
);

App.propTypes = {
  EventsCmpt: PropTypes.func,
};

App.defaultProps = {
  EventsCmpt: Events,
};

export default App;
