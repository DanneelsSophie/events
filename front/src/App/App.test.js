import { render } from '@testing-library/react';
import App from './App';

describe('<App/>', () => {
  test('should app render without crashing', () => {
    const Events = () => 'Events';
    const { baseElement } = render(<App EventsCmpt={Events} />);
    expect(baseElement).toMatchSnapshot();
  });
});
