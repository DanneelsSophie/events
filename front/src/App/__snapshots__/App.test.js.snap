// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> should app render without crashing 1`] = `
<body>
  <div>
    Events
  </div>
</body>
`;
